project/ws/app/src/lib/routes/home/routes/events/edit-event/edit-event.component.ts

```diff
@@ -174,7 +177,11 @@ export class EditEventComponent implements OnInit {
         const eTime = new Date(newendDate).valueOf()
         const cDate = new Date().valueOf()
         if (eTime < cDate) {
-          this.createEventForm.controls['conferenceLink'].setValue(eventObj.recordedLinks[0])
+          if (eventObj.recordedLinks && eventObj.recordedLinks.length > 0) {
```

Comment:
There should be no need to check length > 0 , eventObj.recordedLinks.length will be ok